tests/unit_tests/test_data_photon.py

```diff
@@ -144,3 +144,11 @@ def test_export_to_hdf5(tmpdir, element):
            element2.bremsstrahlung['electron_energy']).all()
     # Export to hdf5 again
     element2.export_to_hdf5(filename, 'w')
+
+def test_photodat_only(tmpdir):
```

Comment:
If you use the `run_in_tmpdir` fixture, the test will execute in a temporary directory without you having to do anything
```suggestion
def test_photoatomic_data_only(run_in_tmpdir):
```